Sources/momento/auth/CredentialProvider.swift

```diff
@@ -6,15 +6,24 @@ enum CredentialProviderError: Error {
     case badToken
 }
 
+/// Specifies the fields that are required for a Momento client to connect to and authenticate with the Momento service.
 public protocol CredentialProviderProtocol {
+    /// API key provided by user, required to authenticate with the Momento service
     var originalAuthToken: String { get }
+    
+    /// parsed API key
```

Comment:
I am not grokking the diff between the `originalAuthToken` and the `authToken`

Reply:
Should these also not be `apiKeys`? Looks like we have moved most of the language over from authToken to apiKey